corehq/apps/export/const.py

```diff
@@ -21,3 +27,5 @@
 
 FORM_EXPORT = 'form'
 CASE_EXPORT = 'case'
+
+MAIN_TABLE = None
```

Comment:
@benrudolph 
I've been using `[]` to designate the table based on the root of the document, and the document traversing code currently makes the same assumption. Is it easy to change this to `MAIN_TABLE = []`? (I'm guessing https://github.com/dimagi/commcare-hq/pull/10199/ does make it easy?)